mongo.go

```diff
@@ -2,15 +2,16 @@ package cachego
 
 import (
 	"errors"
+	"time"
+
 	"gopkg.in/mgo.v2"
 	"gopkg.in/mgo.v2/bson"
-	"time"
 )
 
 type (
 	// Mongo it's a wrap around the mgo driver
 	Mongo struct {
-		collection *mgo.Collection
+		Collection *mgo.Collection
```

Comment:
Well, changing the variable unexported type to exported it's a good approach, I recommend you to create a method for this job, like the NewMapCache.

Maybe NewMongo() function.